.rspec

```diff
@@ -1,3 +1,2 @@
 --color
 --require spec_helper
---require appveyor/worker
```

Comment:
Won't this break the AppVeyor build?

Reply:
I don't work much with Appveyor anymore, but the build passed.  Note that GH Actions runs ten concurrent Windows/Ubuntu jobs, and five concurrent macOS jobs.

It's not needed here, but Actions sets up Windows Ruby build environments without needing any script...